guidelines/MAKE_AN_EXAMPLE.md

```diff
@@ -0,0 +1,56 @@
+# Making an example
+
+> **&lt;short component description&gt;**
+
+## Table of Contents
+
+1. [Examples folder](#examples)
+2. [Change example](#change)
+3. [Angular.json](#angularjson)
+4. [Ng-package.examples.json](#ngpackageexamplejson)
+5. [whitelistedNonPeerDependencies](#whitelistedNonPeerDependencies)
+6. [Npm](#npm)
+
+---
+
+## [Examples folder](#examples)
+
+Copy the entire `examples` folder situated under `examples_folder`.
+
+## [Change example](#change)
+
+Change all instances from `avatar` to your components name under the `examples` folder.
+
+## [Angular.json](#angularjson)
```

Comment:
Only an issue for existing components. New components should get their config and folder structure automatically.